dataplane/ebpf/src/bindings.rs

```diff
@@ -1,4 +1,4 @@
-/* automatically generated by rust-bindgen 0.63.0 */
+/* automatically generated by rust-bindgen 0.61.0 */
```

Comment:
Seems like we should continue using the latest release, even if it doesn't seem to make a difference? I don't think it should change anything below but might as well right?